polyfill/lib/casts.mjs

```diff
@@ -129,10 +99,28 @@ export function CastDate(arg, aux) {
   throw new RangeError(`invalid date ${arg}`);
 }
 
+export function TimeFromString(arg, Construct) {
```

Comment:
super minor, but "Construct" seems like it's a verb, I'd expect "Constructor" or "C" here